install/deploy/base/rhods-application.yaml

```diff
@@ -0,0 +1,207 @@
+---
+apiVersion: applications.console.openshift.io/v1alpha1
+kind: RhodsApplication
+metadata:
+  name: anaconda-ce
+spec:
+  displayName: Anaconda Commercial Edition
+  provider: Anaconda
+  description: |-
+    The world's most popular open-source package distribution and management experience. Optimized
+    and supported for commercial use.
+  kfdefApplications: []
+  route: ''
+  img: images/anaconda-ce.svg
+  category: Partner managed
+  support: third party support
+  docsLink: https://docs.anaconda.com/
+  quickStart: ''
+  getStartedLink: https://anaconda.cloud/register?utm_source=redhat-rhods-summit
+  enable:
+    title: Connect Anaconda to JupyterHub
+    actionLabel: Connect
+    description: ''
+    variables:
+      Anaconda_ce_key: password
+    variableDisplayText:
+      Anaconda_ce_key: Anaconda CE Key
+    variableHelpText:
+      Anaconda_ce_key: This key is given to you by Anaconda
+    validationJob: anaconda-ce-periodic-validator
+    validationSecret: anaconda-ce-access
+    validationConfigMap: anaconda-ce-validation-result
+---
+apiVersion: applications.console.openshift.io/v1alpha1
+kind: RhodsApplication
+metadata:
+  name: jupyterhub
+spec:
+  displayName: JupyterHub
+  provider: Jupyter
+  description: |-
+    A multi-user version of the notebook designed for companies, classrooms and research labs.
+  kfdefApplications: ['jupyterhub', 'notebook-images']
+  csvName: opendatahub
+  route: jupyterhub
+  routeNamespace: redhat-ods-applications
```

Comment:
No need to set the `routeNamespace` here, should only be specified if it will always be in a particular namespace (not the same namespace as the dashboard).

Reply:
done